diff --git a/data/retroshare.desktop b/data/retroshare.desktop
index efc73dd6b..329ed6dc5 100644
--- a/data/retroshare.desktop
+++ b/data/retroshare.desktop
@@ -3,7 +3,7 @@ Encoding=UTF-8
 Version=1.0
 Name=RetroShare
 Comment=Securely communicate with your friends
-Exec=/usr/bin/retroshare %U
+Exec=/app/bin/retroshare -t /app/bin/tor %U
 Icon=/usr/share/pixmaps/retroshare.xpm
 Terminal=false
 Type=Application
